index.html

```diff
@@ -257,6 +258,13 @@ <h4>The Top-Level <code>&lt;math&gt;</code> Element</h4>
             attribute is absent or has an invalid value, the User Agent
             stylesheet treats it the same as <code>inline</code>.
           </p>
+
+            
+	  <p>Systems implementing MathML Core may ignore
+	  <dfn><code>alttext</code></dfn>, however it is used by as
+	  alternative text by some legacy systems that do not
+	  implement math layout.</p>
```

Comment:
My preference would be to say that MathML Core does not define any behavior for `altext` and add a note (thus non-normative) regarding use by other legacy systems.

Reply:
done